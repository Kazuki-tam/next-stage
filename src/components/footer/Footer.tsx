import type { VFC } from 'react'

import styles from './Footer.module.scss'

export const Footer: VFC = () => {
  return (
    <footer className={styles.footer}>
      <small lang="en">@ 20xx Sample.com</small>
    </footer>
  )
}
