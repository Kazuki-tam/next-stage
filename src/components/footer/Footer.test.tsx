import renderer from 'react-test-renderer'

import { Footer } from './Footer'

it('renders correctly footer', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})
