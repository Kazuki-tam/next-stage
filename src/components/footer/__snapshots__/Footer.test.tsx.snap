// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly footer 1`] = `
<footer
  className="footer"
>
  <small
    lang="en"
  >
    @ 20xx Sample.com
  </small>
</footer>
`;
